pro/src/core/OfferEducational/adapters/patchIsOfferActiveAdapter.ts

```diff
@@ -0,0 +1,37 @@
+import * as pcapi from 'repository/pcapi/pcapi'
+
+type IPayloadSuccess = null
+type IPayloadFailure = null
+type PatchIsOfferActiveAdapter = Adapter<
+  { offerId: string; isActive: boolean },
+  IPayloadSuccess,
+  IPayloadFailure
+>
+
+export const patchIsOfferActiveAdapter: PatchIsOfferActiveAdapter = async ({
+  offerId,
+  isActive,
+}) => {
+  try {
+    await pcapi.updateOffersActiveStatus(false, {
+      ids: [offerId],
+      isActive,
+    })
+
+    return {
+      isOk: true,
+      message: isActive
+        ? 'Votre offre est maintenant active et visible dans Adage'
+        : 'Votre offre est maintenant inactive et sera invisible pour les utilisateurs d’adage',
```

Comment:
(typo) 

```suggestion
        : 'Votre offre est maintenant inactive et sera invisible pour les utilisateurs d’Adage',
```

juste pour être homogène avec le message à la ligne du dessus. Ou bien ADAGE, cf. mon commentaire plus haut.